src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListServiceImpl.java

```diff
@@ -90,6 +91,7 @@ public PostDailyListResponse saveDailyListXmlToDatabase(DailyListPostRequest pos
         dailyListEntity.setCourthouse(courthouse);
         dailyListEntity.setXmlContent(postRequest.getDailyListXml());
         dailyListEntity.setSource(postRequest.getSourceSystem());
+        dailyListEntity.setStatus(JobStatusType.NEW.name());
```

Comment:
The status field on the DailyListEntity feels odd here around the different ways we try to setStatus in code / test.  Ideally it would be changed to be something like the enum usage in `MediaRequestEntity` to make the code more readable and it looks like really we should be ensuring the column is `not null` as well?

```
    @Column(name = REQUEST_STATUS_COLUMN_NAME, nullable = false)
    @Enumerated(EnumType.STRING)
    private AudioRequestStatus status;
```

Reply:
@HMCTSDeveloper Looks like @barry-jones50410 has made the database change recently in v48 TABLE daily_list.
`job_status                  CHARACTER VARYING             NOT NULL`
Do you think you could add the corresponding flyway script - may need to ask the test team if the dodgy records can be deleted from staging?

Reply:
ACCEPT, will create separate jira for Barry's changes.